src/policy/fees.h

```diff
@@ -159,6 +171,10 @@ class CBlockPolicyEstimator
 
 class FeeFilterRounder
 {
+private:
+    static constexpr double MAX_FILTER_FEERATE = 1e7;
+    static constexpr double FEE_FILTER_SPACING = 1.1;
```

Comment:
yeah, there is no relation.
when I made the filter spacing, I just copied the other one arbitrarily, but then I decided if I was changing the estimate spacing, there was no reason to also make a change to the filter spacing.   I can add a comment.